### ISSUE TITLE
setup.py install broken over python version

### ISSUE THREAD

**Issue by marcobergman** [pypilot/pypilot#77]
pi@openplotter:~/pypilot $ sudo python3 setup.py install
[...]
```
Extracting pypilot-0.24-py3.7-linux-armv7l.egg to /usr/local/lib/python3.7/dist-packages
  File "/usr/local/lib/python3.7/dist-packages/pypilot-0.24-py3.7-linux-armv7l.egg/pypilot/failedimports.py", line 14
    print 'pypilot requires python version 3.  python version is', sys.version
                                                                ^
SyntaxError: Missing parentheses in call to 'print'. Did you mean print('pypilot requires python version 3.  python version is', sys.version)?
```

**Comment by marcobergman** [comment]
Pull request #78

**Comment by seandepagnier** [comment]
something else is wrong...  that code is not supposed to execute under python3

**Comment by marcobergman** [comment]
Ok this happened on a spanking new openplotter image; I installed pypilot through the openplotter menu and got v0.16, while the opencpn plugin is 0.22. That did not talk to each other so I had to reinstall pypilot from github. But the setup.py failed. At some point I changed the symbolic link /usr/bin/python to point to python3, but I believe that that did not solve the problem with setup.py. This pull request did.
